.travis.yml

```diff
@@ -112,6 +115,14 @@ script:
     - $CXX --version
     - if [ -z "$TRAVIS_TAG" ]; then cargo build --verbose; fi
     - if [ -z "$TRAVIS_TAG" ]; then cargo test -- --nocapture; fi
+    - |
+      if [ -z "$COVERAGE" ]; then
+        cargo clean;
+        CXXFLAGS="--coverage" LDFLAGS="--coverage -lgcov" CARGO_INCREMENTAL=0 RUSTFLAGS="-Zprofile -Ccodegen-units=1" cargo +nightly test;
```

Comment:
We can run the tests with these directly, so we don't need to run the tests twice